index.html

```diff
@@ -57,6 +57,172 @@ <h1 id="head-1">
+          </div>
+        </div>
+        <!---------proj-card--------- -->
+
+        <div class="proj-card">
+          <img src="imege/web-imege01.webp" />
+
+          <div class="card-content">
+            <h2>
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h2>
+            <ul class="tools">
+              <li>Ruby on rails</li>
+              <li>Css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="See-Project">
+              <a href="#"></a>See Project
+            </button>
+          </div>
+        </div>
+        <!---------proj-card--------- -->
+
+        <div class="proj-card">
+          <img src="imege/web-imege01.webp" />
+
+          <div class="card-content">
+            <h2>
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h2>
+            <ul class="tools">
+              <li>Ruby on rails</li>
+              <li>Css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="See-Project">
+              <a href="#"></a>See Project
+            </button>
+          </div>
+        </div>
+        <!---------proj-card--------- -->
+
+        <div class="proj-card">
+          <img src="imege/web-imege01.webp" />
+
+          <div class="card-content">
+            <h2>
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h2>
+            <ul class="tools">
+              <li>Ruby on rails</li>
+              <li>Css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="See-Project">
+              <a href="#"></a>See Project
+            </button>
+          </div>
+        </div>
+        <!---------proj-card--------- -->
+
+        <div class="proj-card">
+          <img src="imege/web-imege01.webp" />
+
+          <div class="card-content">
+            <h2>
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h2>
+            <ul class="tools">
+              <li>Ruby on rails</li>
+              <li>Css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="See-Project">
+              <a href="#"></a>See Project
+            </button>
+          </div>
+        </div>
+        <!---------proj-card--------- -->
+      </div>
+
+      <!-- -----------About Section----------------- -->
+
+      <div class="about">
+        <h2>About me</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button type="button" class="My-Resume">
+          <a href="#"></a>Get My Resume
+        </button>
+
```

Comment:
- There are some issues in this section that need your attention.
- [ ] Your about section is currently lacking the background images. It would be good to add that so as to follow the template design.
- [ ] Kindly use the correct font for your heading and paragraph as in Figma.
- [ ] Kindly style your button properly and implement the various button interactions 💯 
- Your design:
![Annotation 2022-03-18 202645](https://user-images.githubusercontent.com/85113567/159077242-3d9d7628-e794-4eee-855b-606acf696b66.jpg)
- Figma design:
![Annotation 2022-03-18 202704](https://user-images.githubusercontent.com/85113567/159077262-bdfa284e-8e03-4c79-88d0-65a559ebec2f.jpg)